### PR TITLE
fix!: definition list improvements for wrapping and alignment

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

At the moment the label is cut off after 156px in horizontal layouts. Also, the justified option actually right aligns the text 🤔

## 🚀 Changes

Refactored the markup a bit. Moved CSS into the component. Deprecated the CSS in Neptune CSS. Increased the label width to 30%.

**BREAKING CHANGES:** I changed the HORIZONTAL_JUSTIFIED layout from right aligning to actually justifying, and I added a new HORIZONTAL_RIGHT_ALIGNED. Also, long labels will now wrap, not truncate. 

## ✅ Checklist

- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,14 +114,14 @@
...
-        2,
+        1,
         "never"
       ],
       "type-case": [
         0
       ],
       "type-empty": [
-        2,
+        1,
```
Comment (dozortsev) [inline package.json:124]: what this does?

File: packages/components/.storybook/config.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ import { withInfo } from '@storybook/addon-info';
 import { withA11y } from '@storybook/addon-a11y';
 
 import theme from './common/theme';
-import 'bootstrap/dist/css/bootstrap.css';
+import '@transferwise/neptune-css/dist/css/neptune.css';
```
Comment (dozortsev) [inline packages/components/.storybook/config.js:8]: 🙇‍♂️

File: packages/css/src/less/core/_scaffolding.less [hunk 3]
```diff
@@ -286,6 +287,7 @@ dd {
   }
 }
 
+/* Deprecated - use the Neptune Web DefinitionList component */
```
Comment (dozortsev) [inline packages/css/src/less/core/_scaffolding.less:290]: why can't we just delete this code?

File: postcss.config.js [hunk 4]
```diff
@@ -9,7 +9,9 @@ module.exports = {
     'postcss-import': {},
     autoprefixer: {},
     'postcss-custom-properties': {},
-    'postcss-custom-media': {},
+    'postcss-custom-media': {
+      stage: 1,
```
Comment (dozortsev) [inline postcss.config.js:13]: what this for? :)

File: package.json (hunk 1 above)
Comment (helendurranttw) [reply package.json:124]: Ugggh, the commit lint started complaining about the commits in the branch with !s in them, even though we were under the impression they were valid :( So I've moved them to warnings for now. [This is the relevant issue](https://github.com/conventional-changelog/commitlint/issues/878).

File: packages/css/src/less/core/_scaffolding.less (hunk 3 above)
Comment (helendurranttw) [reply packages/css/src/less/core/_scaffolding.less:290]: We agreed to give people the chance to remove their use of deprecated components (or classes) for a while after deciding to deprecating them, before we remove them.

**Pull request merged**